README.md

```diff
@@ -152,26 +151,27 @@ Update entry with ID `654651caf696083cab72ab1c` in the `events` collection:
 -   **Endpont:** `/user/editUserInfo/:id`
     Headers: `Content-Type: application/json`
     Body: JSON-formatted data
-    Changes: Only optional data of User(firstName, lastName, region, organizationName, phone). Admin can change also "isDeleted" and "role" property.
+    Changes: Only optional data of User(firstName, lastName, region, organizationName, phone).Admin and User itself can change the data, but only if the requester of changes is not deleted or is approved.
```

Comment:
Why deleted space before the start of the sentence "Admin..." ?

Reply:
Fixed